.azure/templates/run-qns.yml

```
@@ -0,0 +1,50 @@
+#
+# This pipeline executes a single run of the interop runner between a chosen
+# client and server implementation.
+#
+
+jobs:
+  - ${{ each client in parameters.clients }}:
+    - ${{ each server in parameters.servers }}:
+      - job:
+        displayName: ${{ client }} -> ${{ server }}
+        variables:
+          runCodesignValidationInjection: false
+        steps:
+        - script: |
+            sudo apt install -y cmake libglib2.0-dev libgcrypt20-dev flex bison \
```

`apt` is not recommended for non-interactive use. Try:
```suggestion
            sudo apt-get install -y cmake libglib2.0-dev libgcrypt20-dev flex bison \
```